src/base/bittorrent/session.h

```diff
@@ -418,6 +418,13 @@ namespace BitTorrent
         OSMemoryPriority getOSMemoryPriority() const;
         void setOSMemoryPriority(OSMemoryPriority priority);
 #endif
+        // Peer Turnover
```

Comment:
These seem to be randomly placed, comment is also unnecessary here.

Reply:
Happy to place them wherever you think is sensible. Please advise. I can lose the comment.

Reply:
I will place them place them below `void setAnnounceToAllTiers(bool val);` as advised by @Chocobo1